examples/04_manipulating_images/plot_compare_mean_image.py

```diff
@@ -0,0 +1,27 @@
+"""
+Comparing the means of 2 images
+===============================
+
+Here we compare the means of 2 resting state 4D images.
```

Comment:
Say in the docstring that the goal of this example is to demonstrate the use of "math_img". You should also mention that "mean_img" could have been used to compute the means.